README.md

```diff
@@ -1 +1,19 @@
-# bouncerscript
+Bouncerscript README
+
+[![Build Status](https://travis-ci.org/mozilla-releng/bouncerscript.svg?branch=master)](https://travis-ci.org/mozilla-releng/bouncerscript)
+[![Coverage Status](https://coveralls.io/repos/github/mozilla-releng/bouncerscript/badge.svg?branch=master)](https://coveralls.io/github/mozilla-releng/bouncerscript?branch=master)
+
+
+## deploy a new version
+
+1. Once your PR is reviewed and passes the tests, have one of the admins review & merge it
+2. Bump to new version in `version.json` and commit with a "%VERSION%" message
+3. `git tag -s %VERSION%`
+4. `git push`
+5. `git push --tags`
+6. Create tarball with `python setup.py sdist` and scp that file under [puppet](http://releng-puppet2.srv.releng.scl3.mozilla.com/python/packages-3.5/)
+7. Wait for that file to be synchronized across all puppet instances (emails arrive to confirm that)
+8. Tweak the `bouncerscript` version under [bouncerworker module](https://hg.mozilla.org/build/puppet/file/tip/modules/bouncer_scriptworker/manifests/init.pp#l29) to reflect the new value
```

Comment:
Nit: The line number may not be valid anymore of looking at the tip file. I'd recommend to `s/tip/some_known_rev`

Reply:
DXR permalink FTW https://dxr.mozilla.org/build-central/rev/a5b360575c6f6b67b1093b81145f4700b13bd9da/puppet/modules/bouncer_scriptworker/manifests/init.pp#29 👍